src/sentry/api/endpoints/auth_index.py

```diff
@@ -26,6 +27,14 @@
 
 PREFILLED_SU_MODAL_KEY = "prefilled_su_modal"
 
+DISABLE_SSO_CHECK_SU_FORM_FOR_LOCAL_DEV = getattr(
```

Comment:
I think having this outside of `AuthIndexEndpoint` means these constants will be filled out at module load and can't be refreshed unless there's a restart.

Reply:
I think currently, we can modify the config and _validate_authentication will pick it up.

Reply:
im not sure if the settings will change without a restart

Reply:
If the config is already loaded to memory on startup, then yeah, this will work.